public/scripts/i18n.js

```diff
@@ -112,11 +112,17 @@ export function applyLocale(root = document) {
                 if (localizedValue || localizedValue == '') {
                     $(this).attr(attributeMatch[1], localizedValue);
                 }
+                else {
+                    console.warn(`Missing translation for '${attributeMatch[2]}'`);
```

Comment:
That is gonna cause a lot of console spam

Reply:
According to my experiments, there are not many warnings
I personally think this helps translators and developers figure out the problem, what do you think?

Reply:
I think I can add it back anytime, so [`822bcc6` (#2382)](https://github.com/SillyTavern/SillyTavern/pull/2382/commits/822bcc696039fe540d39f55795fe9f65a4111858) fixed

Reply:
Similar tools are in the debug menu and i18n repo, so this is redundant